test/validation/api/dma/dma.c

```diff
@@ -1108,7 +1108,7 @@ static void test_dma_pkt_segs_to_addr_sync(void)
 	uint8_t *dst;
 	odp_packet_t pkt;
 	uint32_t i, len, num_segs;
-	uint32_t pkt_len = global.pkt_len;
+	uint32_t pkt_len = ODPH_MIN(global.pkt_len, global.len);
```

Comment:
The following if statement below seems now unnecessary:
```
	if (len > global.len)
		len = global.len;
```

Otherwise looks OK.